mantidproject/mantid
# Adding Project Recovery concepts docs

Creating concepts page for project recovery. Addresses #22953.

**Report to:** Nick Draper

**To test:**

Look at the documentation, make sure that it covers all relevant points


Does this update require release notes?

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
